heaps/min_heap.py

```diff
@@ -44,33 +53,57 @@ def __str__(self):
 
     def empty(self):
         """ This method returns true if the heap is empty
-            Time complexity: ?
-            Space complexity: ?
+            Time complexity: O(1)
+            Space complexity: O(1)
         """
-        pass
+        return len(self.store)== 0
 
 
     def heap_up(self, index):
         """ This helper method takes an index and
-            moves the corresponding element up the heap, if 
+            moves the corresponding element up the heap, if
             it is less than it's parent node until the Heap
             property is reestablished.
-            
+
             This could be **very** helpful for the add method.
             Time complexity: ?
             Space complexity: ?
```

Comment:
✨ Very clean. Time and space complexity?

Reply:
I added the following:
Time Complexity:  O(log n)
Space Complexity: O(log n)
Thanks!